CMakeLists.txt

```diff
@@ -212,10 +212,25 @@ if(NOT DISABLE_DEPENDENCY_VERSION_CHECKS)
   set(MINIMUM_DOXYGEN_VERSION 1.8.8)
 endif()
 
+# VFX 18 deprecations to transition to MINIMUM_* variables in OpenVDB 7.0.0
+
+set(DEPRECATED_CMAKE_VERSION 3.12)
+set(DEPRECATED_GCC_VERSION 6.3.1)
+set(DEPRECATED_ICC_VERSION 17)
+set(DEPRECATED_MSVC_VERSION 2017)
+set(DEPRECATED_TBB_VERSION 2017.6)
+set(DEPRECATED_NUMPY_VERSION 1.12.1)
+set(DEPRECATED_HOUDINI_VERSION 17)
```

Comment:
Should Maya also be here? Note that MSVC is "2017 15.0", I'm not sure exactly how the version is set in CMake. Based on @danrbailey's reply r.e. default versions with certain OS distributions, I've left GLFW and log4-cplus alone.

Reply:
After looking at the appveyor build it looks like I'll need to change the MSVC version

Reply:
Could these variables perhaps be named "`FUTURE_MINIMUM_*_VERSION`"?

Reply:
done